### PR TITLE
Disable welcome message in non-interactive mode

### PR DESCRIPTION
**Pull request by adtac**
Closes https://github.com/coala/coala-quickstart/issues/51

### CODE REVIEW

File: coala_quickstart/coala_quickstart.py [hunk 1]
```diff
@@ -49,7 +49,8 @@ def main():
     printer = ConsolePrinter()
     log_printer = LogPrinter(printer)
 
-    print_welcome_message(printer)
+    if not args.non_interactive:
+        print_welcome_message(printer)
```
Comment (sils) [inline coala_quickstart/coala_quickstart.py]: whynot simply move this below line 56?

Comment (adtac) [inline coala_quickstart/coala_quickstart.py]: lol yeah that's better, fixed :+1:

**Comment by Nosferatul** [comment]
ack 6331560

**Comment by sils** [comment]
@rultor merge

**Pull request merged**